### DISCUSSION TITLE
Disabling preview deploys from forked repos

### DISCUSSION THREAD

**Discussion by kochis** [vercel/community#442] [Help]
Hi there,

Is there a way / best practice to prevent preview deploys from running on a forked repo?

We're using previews for a public documentation repo, but would like to keep the ability to deploy previews just for members of our organization. 

I noticed that `Git Fork Protection` requires approval for changes to the Vercel config: https://vercel.com/docs/concepts/git#deploying-forks-of-public-git-repositories, but is there any way to prevent the forked deploys from creating previews altogether?

Thanks.

**Comment by samcx** [comment]
"there any way to prevent the forked deploys from creating previews altogether?" → To clarify, are you referring to not getting a notification of a forked repo altogether?

**Reply by kochis** [reply]
Not sure what you mean by notification, but if someone outside our organization forks our repo, it would be great if we could prevent the pull requests that from _from_ the fork, to not generate preview.

We basically want to avoid letting _anyone_ create a preview app in our environment from the codebase, and only members of our organization.

**Reply by samcx** [reply]
@kochis Can you clarify if you are using a Vercel team?
